conda_smithy/templates/azure-pipelines-win.yml.tmpl

```
@@ -56,9 +56,9 @@ jobs:
 {%- endif %}
     - task: CondaEnvironment@1
       inputs:
-        packageSpecs: 'python=3.9 conda-build conda pip {{ BOA }}{{ " ".join(remote_ci_setup) }}' # Optional
-        installOptions: "-c conda-forge"
-        updateConda: true
+        packageSpecs: 'python=3.9 conda-build conda=4.12 pip {{ BOA }}{{ " ".join(remote_ci_setup) }}' # Optional
+        installOptions: "-c conda-forge --strict-channel-priority"
```

```suggestion
        installOptions: "-c conda-forge --override-channels"
```

Can we not drop defaults early in the setup process? I thought we are quite independent from them now.